src/main/java/com/stripe/model/BankAccount.java

```diff
@@ -12,13 +19,25 @@
 	String fingerprint;
 	Boolean defaultForCurrency;
 
-	public String getId() {
-		return id;
-	}
-
-	public void setId(String id) {
-		this.id = id;
-	}
+    public BankAccount update(Map<String, Object> params)
+            throws AuthenticationException, InvalidRequestException, APIConnectionException, CardException, APIException {
+        return update(params, (RequestOptions) null);
+    }
+
+    public BankAccount update(Map<String, Object> params, RequestOptions options)
+            throws AuthenticationException, InvalidRequestException, APIConnectionException, CardException, APIException {
+        return request(RequestMethod.POST, this.getInstanceURL(), params, BankAccount.class, options);
+    }
+
+    public DeletedBankAccount delete()
```

Comment:
had to go back to java 101 and google for this http://stackoverflow.com/questions/14519797/overriding-a-method-with-different-return-types-in-java to remember that it works, but bullish bullish :)